kzz7yh-f19041/cod-ve09v2_kzz7yh-f19041.xml

```diff
@@ -107,7 +108,7 @@
    </p>
                <p xml:id="kzz7yh-f19041-d1e185">
                   <lb ed="#V" n="5"/>Respondeo quod lucifer inter omnes angelos fuit 
-      <lb ed="#V" n="6"/>creatus nobilior. Et si stetisset: et ad 
+      <lb ed="#V" n="6"/>creatus nobilior. Et si stetisset<!--TODO Abbreviation-->: et ad 
```

Comment:
```suggestion
      <lb ed="#V" n="6"/>creatus nobilior. Et si stetisset: et ad 
```

Reply:
I don't think there is an abbreviation after "Stetisset," I think it's just a colon and then the abbreviaiton for "et."